patches/minecraft/net/minecraft/world/level/newbiome/layer/RegionHillsLayer.java.patch

```diff
@@ -0,0 +1,15 @@
+--- a/net/minecraft/world/level/newbiome/layer/RegionHillsLayer.java
++++ b/net/minecraft/world/level/newbiome/layer/RegionHillsLayer.java
+@@ -51,7 +_,11 @@
+       } else {
+          if (p_76838_.m_5826_(3) == 0 || k == 0) {
+             int l = i;
+-            if (i == 2) {
++            java.util.Optional<net.minecraft.resources.ResourceKey<net.minecraft.world.level.biome.Biome>> hills = net.minecraftforge.common.BiomeManager.getHillsBiome(((net.minecraftforge.registries.ForgeRegistry<net.minecraft.world.level.biome.Biome>)net.minecraftforge.registries.ForgeRegistries.BIOMES).getKey(i));
++            if(hills.isPresent()) {
++               l = net.minecraft.data.BuiltinRegistries.f_123865_.m_7447_(net.minecraft.data.BuiltinRegistries.f_123865_.m_6246_(hills.get()));
++            }
++            else if (i == 2) {
```

Comment:
This probably should be: 
```java
} else
if (i == 2) {
```
So as to not have this patch having to remove any lines and just add some.